README.md

```diff
@@ -45,11 +45,11 @@ Let's make a planetary system!
     - Create reader methods to give a user access to read the instance variables.
 
 ## Optionals
--  Create a method which creates a planet from user input.
+-  Create a method, outside any class, which creates a planet from user input.
 
 # Wave 3
 ## Primary Requirements
-- Create an interface where the user can select a planet and be presented information about the planet.  
+- Create an interface where the user can select a planet and be presented information about the solar system and select a planet to view more information about.  
```

Comment:
I feel like this sentence reads weirdly, especially the repetition of "select a planet".